controllers/authors.js

```diff
@@ -0,0 +1,41 @@
+const models = require('../models')
+
+const getAllAuthors = async (request, response) => {
+  try {
+    const authors = await models.Authors.findAll()
+
+    return response.send(authors)
+  } catch (error) {
+    return response.status(500).send('Unable to retrieve authors list, please try again.')
+  }
+}
+
+const getAuthorByIdentifier = async (request, response) => {
```

Comment:
This route throws an error

![image](https://user-images.githubusercontent.com/5775716/82481356-2a679680-9aa3-11ea-8d69-1e1f96c9f424.png)